source/shared/cpp/ObjectModel/BaseActionElement.cpp

```diff
@@ -86,17 +85,16 @@ Json::Value BaseActionElement::GetAdditionalProperties() const
     return m_additionalProperties;
 }
 
-void BaseActionElement::SetAdditionalProperties(Json::Value const &value)
+void BaseActionElement::SetAdditionalProperties(Json::Value const& value)
 {
     m_additionalProperties = value;
 }
 
 void BaseActionElement::PopulateKnownPropertiesSet()
 {
-    m_knownProperties.insert({ AdaptiveCardSchemaKeyToString(AdaptiveCardSchemaKey::Type),
-         AdaptiveCardSchemaKeyToString(AdaptiveCardSchemaKey::Title),
-         AdaptiveCardSchemaKeyToString(AdaptiveCardSchemaKey::Id),
-         AdaptiveCardSchemaKeyToString(AdaptiveCardSchemaKey::IconUrl)});
+    m_knownProperties.insert({AdaptiveCardSchemaKeyToString(AdaptiveCardSchemaKey::Type),
+        AdaptiveCardSchemaKeyToString(AdaptiveCardSchemaKey::Title), AdaptiveCardSchemaKeyToString(AdaptiveCardSchemaKey::Id),
```

Comment:
Here too, at the caller. This seems less readable/obvious with the long arguments sharing a line.